flink-libraries/flink-table/src/test/java/org/apache/flink/table/api/java/batch/sql/SqlITCase.java

```diff
@@ -18,9 +18,15 @@
 
 package org.apache.flink.table.api.java.batch.sql;
 
+import com.google.common.collect.ImmutableMap;
```

Comment:
Can we use `java.util.Collections.singletonMap()` instead of Guava? We avoid using `Guava` due to version conflicts.